docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    In our :ref:`user guide <user_guide>` we provide links to the papers describing
+    Fairlearn's algorithms.
+    For a higher level view of the subject, please see the :ref:`further reading <further_reading>`
+    section.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
```

Comment:
Source?
I think this is the typical "fairness through awareness", and the paper should have that name (?)

Reply:
The next paragraph explains why it doesn't work. However, I had added that Dwork et al. paper to the further reading section.

Reply:
You can reference papers quite nicely in ReST.

Reply:
I'm sure. That was why I asked about the BibTeX package elsewhere.

Reply:
I see. You don't need BibTeX for that. Just check the fairness_in_machine_learning file for examples. They reference papers as [#1]_, [#2]_, etc.

Reply:
Since discovering BibTeX, I've not written a single `\bibitem` myself. I would prefer not to start now. Moreover, if we reference the same paper on multiple pages, then we should be using BibTeX (or the Sphinx facsimile thereof) simply as good practice. We can guarantee a uniform style, and that there aren't any typos in the references.

It would be a subject for a different PR, though.